test/e2e/helpers/channel_event_tranformation_test_helper.go

```diff
@@ -50,17 +50,26 @@ func EventTransformationForSubscriptionTestHelper(t *testing.T,
 		client.WaitForResourcesReadyOrFail(&channel)
 
 		// create transformation pod and service
-		transformedEventBody := fmt.Sprintf("eventBody %s", uuid.NewUUID())
-		eventAfterTransformation := cloudevents.New(
-			fmt.Sprintf(`{"msg":%q}`, transformedEventBody),
-			cloudevents.WithSource(senderName),
-		)
-		transformationPod := resources.EventTransformationPod(transformationPodName, eventAfterTransformation)
+		eventAfterTransformation := cloudevents.NewEvent()
+		eventAfterTransformation.SetID("dummy-transformed")
+		eventSource := fmt.Sprintf("http://%s.svc/", senderName)
```

Comment:
Can you move eventSource to the top, near the other "constants" like `recordEventsPodName`?

Reply:
done